business/iter8.go

```diff
@@ -24,19 +25,60 @@ type Iter8Service struct {
 
 func (in *Iter8Service) GetIter8Info() models.Iter8Info {
 	var err error
+	var ps []core_v1.Pod
+	var controllerImgVersion string
+	var analyticsImgVersion string
 	promtimer := internalmetrics.GetGoFunctionMetric("business", "Iter8Service", "GetIter8Info")
 	defer promtimer.ObserveNow(&err)
 
 	conf := config.Get()
 
 	// It will be considered enabled if the extension is present in the Kiali configuration and the CRD is enabled on the cluster
 	if conf.Extensions.Iter8.Enabled && in.k8s.IsIter8Api() {
+		if kialiCache != nil && kialiCache.CheckNamespace("iter8") {
```

Comment:
"iter8" literal should come from the config https://github.com/kiali/kiali/blob/master/config/config.go#L162 
It can be initialized with "iter8" if not present in the Kiali yaml config and we can propagate it in the future in the operator if we want to change it.

Reply:
Here we assume that if iter8 extension is enabled every user has access.
That may be fine for this PR, but in the future we may check if "iter8" namespace is accessible (invoking a GetNamespace() first).

But I'm ok without dealing with that check know, it's not a bad assumption that iter8 should be accessible for now.